### ISSUE TITLE
Не работает в CherryTree 0.38.4 Win10 x64

### ISSUE THREAD

**Issue by small-hamster** [andreevlex/gtksourceview-1c-bsl#1]
Синтаксис "BSL" выбрать позволяет, но подсветка не работает. 
Причем если выбрать, например, Java - подсветка работает.

В чем может быть дело?
![default](https://user-images.githubusercontent.com/36257509/35959725-db62a8b6-0cb7-11e8-9f3a-04e8056ab138.png)

**Comment by ava57r** [comment]
У меня Win 10 нет, протестировать не могу.
Попробовал в линксе через редактор Xed, работает.

А как проверяете?
Я вот сначала создал пустой файл с расширением `*.bsl` и указал явно синтаксис.
Подсветка базовых конструкций языка на русском и английском работает.

**Comment by ava57r** [comment]
@small-hamster Привет удалось победить?

**Comment by xtyfus** [comment]
Решил проблему тем, что в этой строке
`   <style id="operator"          _name="Operator"              map-to="def:operator"/>`
поменял `def:operator` на `def:keyword` и закомментировал строку
`<context ref="def:c-like-comment" style-ref="comment"/>`
После этого работает всё, кроме подсветки комментариев.

В стилях не определен operator, можно попробовать его там объявить.

Комментарии заработали после того, как добавил в  defenitions

```
<context id="comment" style-ref="comment" end-at-line-end="true" class="comment" class-disabled="no-spell-check">
		<start>//</start>

		<include>
		  <context ref="def:in-line-comment"/>
		</include>
</context>
```

и в `include` в конце `<context ref="comment"/>`

**Comment by ava57r** [comment]
PR =) ?

**Comment by small-hamster** [comment]
> PR =) ?

Нет, никакого PR. Все просто: после долгого тестирования CherryTree, пришел к выводу, что он мне не подходит)
